### PR TITLE
Welcome Index

### PR DESCRIPTION
**Pull request by rebeccafae**
## Description

Fix styles so the welcome page is responsive

## Screenshots

It's a long page, you should probably pull down the branch and look at it :-)

### CODE REVIEW

File: app/assets/stylesheets/styles.scss [hunk 1]
```diff
@@ -429,146 +314,71 @@ nav {
...
-//     margin-top: 25px;
-//   }
-// }
+@media (min-width: 992px) {
+  .list-md-inline li {
+    display: list-item;
+  }
+}
+
+
```
Comment (marshallds) [inline app/assets/stylesheets/styles.scss]: there's some blank lines here

File: app/assets/stylesheets/styles.scss [hunk 2]
```diff
@@ -429,146 +314,71 @@ nav {
...
+
+#container {
+  font-family: Helvetica Neue, Helvetica, Sans Serif;
+  color: #9B9B9B;
+  margin: 60px;
+}
+
+#container h1,#container h2 {
```
Comment (marshallds) [inline app/assets/stylesheets/styles.scss]: ws

File: app/assets/stylesheets/styles.scss [hunk 3]
```diff
@@ -429,146 +314,71 @@ nav {
...
+@media (min-width: 992px) {
+  .list-md-inline li {
+    display: list-item;
+  }
+}
+
+
+
+#container {
+  font-family: Helvetica Neue, Helvetica, Sans Serif;
```
Comment (marshallds) [inline app/assets/stylesheets/styles.scss]: `font-family: "Helvetica Neue", Helvetica, sans-serif;`

File: app/assets/stylesheets/styles.scss [hunk 4]
```diff
@@ -67,6 +67,10 @@ h3 {
   &.sectionHead {
     font-size: 3.0rem;
     letter-spacing: 3px;
+
+    @media (min-width: 992px) {
```
Comment (marshallds) [inline app/assets/stylesheets/styles.scss]: extract variable?

File: app/assets/stylesheets/styles.scss [hunk 5]
```diff
@@ -67,6 +67,10 @@ h3 {
   &.sectionHead {
```
Comment (marshallds) [inline app/assets/stylesheets/styles.scss:67]: the `h3.sectionHead`'s under How It Works seem like they have a ridiculous amount of padding.

File: app/assets/stylesheets/styles.scss [hunk 6]
```diff
@@ -429,146 +314,71 @@ nav {
...
+#container h1,#container h2 {
+  color:#1BC49B;
+  font-size: 23px;
+}
+
+a.btn:hover {
+  color: white;
+}
+
+.flush {
```
Comment (marshallds) [inline app/assets/stylesheets/styles.scss:341]: are these in use at all? not a fan of all the !important

Comment (rebeccafae) [reply app/assets/stylesheets/styles.scss:341]: it seems that they are used on the user show and edit pages

**Comment by rebeccafae** [comment]
@marshallds please take another look now, thanks

**Pull request merged**